File: sem4/mpp/mpp-proiect-csharp-vreauladudu/domain/Participant.cs
using System;

namespace mpp_proiect_csharp_vreauladudu.domain
{
    [Serializable]
    public class Participant : Entity<int>
    {
        public String Name { get; set; }
        public int MotorcycleID { get; set; }

        public override string ToString()
        {
            return "ID: " + this.Id + " | Name: " + this.Name + " | MotorcycleId: " + this.MotorcycleID;
        }
    }
}

File: sem4/mpp/mpp-proiect-java-vreauladudu/src/main/java/moto/domain/Participant.java
package moto.domain;

public class Participant extends Entity<Integer>
{
    private String name;
    private Integer motorcycleID;

    public String getName()
    {
        return this.name;
    }

    public void setName(String name)
    {
        this.name = name;
    }

    public Integer getMotorcycleID()
    {
        return this.motorcycleID;
    }

    public void setMotorcycleID(Integer motorcycleID)
    {
        this.motorcycleID = motorcycleID;
    }

    @Override
    public String toString() {
        return "Participant{" +
                "id='" + this.getId() + '\'' +
                ", name='" + this.name + '\'' +
                ", motorcycleID=" + this.motorcycleID +
                '}';
    }
}